src/utilities/theme/role-variants.ts

```diff
@@ -318,41 +318,13 @@ export const roleVariants: RoleVariants = {
       light: {lightness: 98},
       dark: {lightness: 98},
     },
-    {
-      name: 'iconSubduedOnPrimary',
-      description:
-        'For use as a fill color for icons on subdued primary actions. Not for use in icons on navigation and tabs.',
-      light: {lightness: 88},
-      dark: {lightness: 88},
-    },
-    {
-      name: 'iconDisabledOnPrimary',
-      description:
-        'For use as a fill color for icons on disabled primary actions. Not for use in icons on navigation and tabs.',
-      light: {lightness: 78},
-      dark: {lightness: 78},
-    },
```

Comment:
These variants are superfluous, as we only style critical and primary buttons with white icons and text, regardless of state.